src/ts/pausedtimeshiftupdatehandler.ts

```diff
@@ -0,0 +1,58 @@
+import { PlayerAPI } from 'bitmovin-player';
+import { Timeout } from './timeout';
+
+export class PausedTimeshiftUpdatedHandler {
+  private static instance: PausedTimeshiftUpdatedHandler;
+  private static _player: PlayerAPI;
+  private pausedTimeshiftUpdater: Timeout;
+  private handlers: CallableFunction[] = [];
```

Comment:
I was more thinking about using our `EventDispatcher` implementation for this 🤔 (sry should have been more clear on that)

Reply:
I mean I can extend it so it uses event dispatcher, but doesn't that just bring in more complexity to this?